src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -116,7 +116,7 @@ export default class ArcanaTracking extends Module {
 		let prepullSleeve = true
 		const sleeveDrawLog: CastEvent[] = []
 		for (const event of events) {
-			if (event.timestamp - startTime >= (STATUSES.SLEEVE_DRAW.duration * 1000)
+		if (event.timestamp - startTime >= (STATUSES.SLEEVE_DRAW.duration * 1000)
```

Comment:
You sure this should be moved?

Reply:
O_O uh tslint? Hewwo?? waht r u doin stahp